### PR TITLE
Profile part for the back-end.

### PR DESCRIPTION
**Pull request by alex2011576**

### CODE REVIEW

File: server/src/repositories/userRepository.ts [hunk 1]
```diff
@@ -63,8 +73,18 @@ const addNewUser = async (newUser: NewUserWithHashedPwd): Promise<User> => {
...
-		values: [newUser.username, newUser.email, newUser.passwordHash, newUser.firstname, newUser.lastname, newUser.activationCode, true, newUser.id42]
+		text: 'insert into users(username, email, password_hash, firstname, lastname, activation_code, is_active, id_42, avatar) values($1, $2, $3, $4, $5, $6, $7, $8, $9) returning *',
+		values: [
+			newUser.username,
+			newUser.email,
+			newUser.passwordHash,
+			newUser.firstname,
+			newUser.lastname,
+			newUser.activationCode,
+			true,
```
Comment (fglsn) [inline server/src/repositories/userRepository.ts:85]: whats that true thing btw?

Comment (fglsn) [reply server/src/repositories/userRepository.ts:85]: a wait nevermind, got it :D

**Review by fglsn — APPROVED** [review]
Looks good, nothing to mention

**Review by KaomN — APPROVED** [review]
Looks good!

**Pull request merged**